lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -183,21 +183,21 @@ export type CoursesResponse = {
 export type Assignment = {
   id: number;
   title: string;
-  course: Course;
+  course: Course; // TODO Optional?
 };
 
 /**
  * Response for `/api/dashboard/assignments/{assignment_id}/stats` call.
  */
-export type StudentStats = {
+export type Student = {
   h_userid: string;
   lms_id: string;
   display_name: string | null;
-  annotation_metrics: AnnotationMetrics;
+  annotation_metrics: AnnotationMetrics; // TODO Optional?
```

Comment:
This is indeed optional in the backend types:

```py
class APIStudent(TypedDict):
    h_userid: str
    """ID of the student in H."""

    lms_id: str
    """ID of the student in the LMS."""

    display_name: str | None

    annotation_metrics: NotRequired[AnnotationMetrics]
```

Reply:
See https://github.com/hypothesis/lms/pull/6453/files#r1684282200